core-bundle/src/Controller/FrontendController.php

```diff
@@ -67,8 +67,6 @@ public function shareAction(): RedirectResponse
     /**
      * Symfony will un-authenticate the user automatically by calling this route.
      *
-     * @throws LogoutException
-     *
```

Comment:
This should stay too IMO. And maybe combined with `@return never`.

Reply:
Added in 4b170582f71bd8906902e2b5fd5ef2205966ae50.